src/index.css

```diff
@@ -0,0 +1,4 @@
+* {
+  margin: 0;
+  font-size: 10px;
+}
```

Comment:
This is styling we want to apply to all components, hence the `*` selector.

A lot of HTML elements automatically have annoying margins. To make it cleaner, we're starting with a blank slate.

Also, there's a huge accessibility concern when it comes to font sizes in particular. It's an accessibility requirement to allow users to resize root font sizes with their browsers. This is helpful for people who have poor eyesight for example, and want to see larger text. Therefore, when we set font sizes, we should use **responsive units** because they change when the user wants them to. `px` are **NOT responsive**; instead, we should use `em` and `rem`. For example:

`h1 {
    font-size: 1.6rem;
}`

`rem` is based on the **root font size**. Therefore, the above code will set h1's font size to _1.6 times the base font size._ Since our base font size is 10px, h1 will render with 16px by default. However, if the user changes their base font size, then it will render at 1.6 times whatever their font size is.

All that to be said... Please divide your desired `px` font size by 10 and use `rem` instead!